akka-persistence-typed/src/main/scala/akka/persistence/typed/scaladsl/PersistentBehavior.scala

```diff
@@ -45,6 +46,16 @@ object PersistentBehavior {
     eventHandler:   (State, Event) ⇒ State): PersistentBehavior[Command, Event, State] =
     PersistentBehaviorImpl(persistenceId, emptyState, commandHandler, eventHandler)
 
+  /**
+   * Create a `Behavior` for a persistent actor that is enforcing replies to commands.
+   */
+  def withEnforcedReplies[Command <: ExpectingReply[_], Event, State](
```

Comment:
Note that the non-enforcing apply remains as an alternative. I don't want to make the enforcement of replies always mandatory because it's not essential for the minimal core API, and it dictates of commands must be defined with the ExpectingReply type.